src/frame.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+class Frame {
+  constructor(number) {
+    this.round = number;
+    this.score = [];
+  }
+
+  addRoll(pins) {
+    this.score.push(pins);
+    return this.score;
+  } 
+
+  spareOrStrike() {
```

Comment:
You're not using this function anywhere. It also doesn't seem very useful given that you already have `isStrike` and `isSpare`.